misc/trash_rolls.txt

```diff
@@ -1,109 +1,122 @@
 title:@chrisfried coming through with a trash list
 description:A must-not-keep list of gear, originally found in top secret group chats.
 
-//notes:🤢🤢🤢
-
 // sidearms (so trash your pal 48klocs can't even be bothered to look them all up)
-dimwishlist:item=-14#notes: Sidearms really warrant the 🤢🤢🤢est of 🤢🤢🤢s.
```

Comment:
Also removed the blanket sidearm thumbs down. Besides no longer being valid, whatever code was put in place to highlight based on item category hashes for sure no longer works at all.

Reply:
Ah - that one was removed from the main list, not the individual list.

Other Discord people have mentioned using ICHes lately and not screamed about it, so that might still actually work?